flaml/model.py

```diff
@@ -786,6 +775,7 @@ def _init_model_for_predict(self):
             Need to reinit training_args because of a bug in deepspeed: if not reinit, the deepspeed config will be inconsistent
             with HF config https://github.com/huggingface/transformers/blob/main/src/transformers/training_args.py#L947
         """
+        self.pred_flag = True

```

Comment:
```suggestion
        self._pred_flag = True
```
Use "_" if for internal use.